core/consensus/common/src/import_queue.rs

```diff
@@ -263,7 +263,7 @@ impl<B: BlockT> BlockImporter<B> {
 		worker_sender: Sender<BlockImportWorkerMsg<B>>,
 		justification_import: Option<SharedJustificationImport<B>>,
 	) -> Sender<BlockImportMsg<B>> {
-		let (sender, port) = channel::unbounded();
+		let (sender, port) = channel::bounded(1);
```

Comment:
You probably want to have a bigger channel. Maybe configurable per CLI.

Reply:
This is way too much of a detail to be configurable per CLI, but +1 for bigger channel.
If the networking task wants to send, say, 4 messages at once, it would have to write one, then switch to the processing task, which reads one, then switch to the network task, to write the next, and so on.